antarest/study/storage/variantstudy/model/command/create_link.py

```diff
@@ -26,22 +44,76 @@
 from antarest.study.storage.variantstudy.model.command_listener.command_listener import ICommandListener
 from antarest.study.storage.variantstudy.model.model import CommandDTO
 
+DEFAULT_COLOR = 112
+FILTER_VALUES: t.List[FilterOption] = [
+    FilterOption.HOURLY,
+    FilterOption.DAILY,
+    FilterOption.WEEKLY,
+    FilterOption.MONTHLY,
+    FilterOption.ANNUAL,
+]
+
+
+def validate_filters(
+    filter_value: Union[List[FilterOption], str], enum_cls: t.Type[FilterOption]
+) -> List[FilterOption]:
+    if filter_value is not None and isinstance(filter_value, str):
+        filter_accepted_values = [e for e in enum_cls]
+
+        options = filter_value.replace(" ", "").split(",")
+
+        invalid_options = [opt for opt in options if opt not in filter_accepted_values]
+        if invalid_options:
+            raise LinkValidationError(
+                f"Invalid value(s) in filters: {', '.join(invalid_options)}. "
+                f"Allowed values are: {', '.join(filter_accepted_values)}."
+            )
+
+        return [enum_cls(opt) for opt in options]
+
+    return filter_value
+
+
+def join_with_comma(values: t.List[FilterOption]) -> str:
+    return ", ".join(value.name.lower() for value in values)
+
+
+class AreaInfo(AntaresBaseModel):
+    area1: str
+    area2: str
+
+
+class LinkInfoProperties(AntaresBaseModel):
+    hurdles_cost: bool = False
+    loop_flow: bool = False
+    use_phase_shifter: bool = False
+    transmission_capacities: TransmissionCapacity = TransmissionCapacity.ENABLED
```

Comment:
`transmission_capacity` in singular (there is an another enum named TransmissionCapacities)

Reply:
The issue is the field is written `transmission-capacities` inside the ini file :/ 
It's a shame that there's another field named the same way inside the generaldata.ini file ...